CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+<!--Markdownlint Rules-->
+<!-- markdownlint-disable no-duplicate-header-->
+
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Changed
+
+- Changed CLI to use CLAP
```

Comment:
Everything looks good. Just make sure that PRs etc are linked 

Example: https://github.com/python-poetry/poetry/blob/1.2.0rc1/CHANGELOG.md?plain=1

Reply:
@John15321 Not all commits had pull requests, but the ones that did i added in.

Reply:
Is poetry autogenerating their content using GitHub releases?

Reply:
If you are talking about their release changelog then yes, I believe so.

Reply:
> Is poetry autogenerating their content using GitHub releases?

The normal changellg, they don't

Reply:
> Is poetry autogenerating their content using GitHub releases?

In the normal changellg, they don't